applications/users/tests/test_user_log_in.py

```diff
@@ -7,13 +7,14 @@
 
 
 class ConfirmedUserLogInTests(APITestCase):
-    url = reverse('rest_login')
     fake = Faker()
 
     def call_log_in(self):
         return self.client.post(self.url, self.params)
 
     def setUp(self):
+        self.url = reverse('rest_login')
+
         self.password = self.fake.password(length=8)
         self.user = UserFactory(password=self.password, confirmed=True)
         self.params = {'email': self.user.email, 'password': self.password}
```

Comment:
Check [`setUpTestData(cls)`](https://docs.djangoproject.com/en/3.0/topics/testing/tools/#django.test.TestCase.setUpTestData). It runs only once before all tests instead of once for each test, I think it would be more appropriate to move the things that won't change there and maybe just let the `self.params = ...` line here.